### PR TITLE
CLOUDP-261756: Repository setup for Postman Project

### PR DESCRIPTION
**Pull request by Luke-Sanderson**
## Proposed changes

Create Postman directory within the repository with openapi and scripts subdirectories.
Add README for the Postman Project
Update dependabot to prepare for dependencies that will be used
Add shellcheck for linting Bash scripts

_Jira ticket:_ CLOUDP-261756

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tools/postman/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+3. **Apply OpenAPI transformations**: Apply transformations such as removing regex to prepare openapi file for
+   conversion.
+
+4. **Convert to Postman Collection**: Use openapi-to-postmanv2 to convert to a Postman Collection.
+
+5. **Apply JSON transformations**: Apply transformations such as changing title, adding fields, and preparing Collection
+   for upload.
+
+6. **Upload Collection to Postman**: Use the Postman API to upload the Collection to Postman.
```
Comment (andreaangiolillo) [inline tools/postman/README.md:56]: could you remove the file `.keep` files from the PR?

File: .github/workflows/shellcheck.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      - "**/**.sh"
+
+jobs:
+  shellcheck:
+    name: shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run ShellCheck
+        uses: bewuethr/shellcheck-action@cdeea6bdd72272a5b839e30edd7efa3f83c424cf
```
Comment (andreaangiolillo) [inline .github/workflows/shellcheck.yml:15]: let's add a new line here 
Context: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: .github/dependabot.yml [hunk 3]
```diff
@@ -19,3 +19,12 @@ updates:
       prefix: "chore"
     reviewers:
       - "mongodb/apix-2"
+  - package-ecosystem: npm
+    directory: "/"
```
Comment (andreaangiolillo) [inline .github/dependabot.yml:23]:
```suggestion
    directory: "/tools/postman"
```

File: tools/postman/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Postman Collections from OpenAPI
+
+Folder contains scripts and other files required to generate a Postman collections from the OpenAPI Specification
```
Comment (andreaangiolillo) [inline tools/postman/README.md:3]:
```suggestion
This folder contains the code required to generate the MongoDB Atlas Postman collections from the MongoDB OpenAPI Specification
```

**Review by andreaangiolillo — APPROVED** [review]
LGTM. Thanks for all the changes 💯

File: .gitignore [hunk 5]
```diff
@@ -15,3 +15,4 @@
 *.vscode
 
 *.out
+*.keep
```
[the remark this answers is not in the document]
Comment (Luke-Sanderson) [reply .gitignore:18]: 👀 👀 I am so used to my IDE adding it on save

**Review by matt-condon — APPROVED** [review]
LGTM

File: tools/postman/README.md (hunk 1 above)
Comment (wtrocki) [reply tools/postman/README.md:57]: Idea is to have gitignore for the entire folder that will keep .keep files forever.

File: .github/dependabot.yml [hunk 6]
```diff
@@ -19,3 +19,12 @@ updates:
       prefix: "chore"
     reviewers:
       - "mongodb/apix-2"
+  - package-ecosystem: npm
```
Comment (wtrocki) [inline .github/dependabot.yml:22]: Q: would postman integration have any files outside /tools/postman ?

**Pull request merged**